src/views/utils.js

```diff
@@ -25,32 +25,23 @@ export function makeBoundingBox(viewState) {
 }
 
 /**
- * Create an initial view state that centers the image in the view at the right zoom level.
- * @param {loader} Object The loader of the image for which the view state is desired.
- * @param {viewSize} Object { height, width } object for deducing the right zoom level to center the image.
+ * Create an initial view state that centers the image in the view at the zoom level that fills the screen.
+ * @param {Object} loader The loader of the image for which the view state is desired.
+ * @param {Object} viewSize { height, width } object for deducing the right zoom level to center the image.
+ * @param {Object} zoomBackOff An positive number for controls how far zoomed out from "full screen" it should be (default is 0).
```

Comment:
hmm. Not in love with this variable name. It took me a little while to figure out how it works (and I'm still not sure if I do). By default, this function returns a `viewState` where the image is scaled to the full viewport, correct? So first we get the `zoom` that fits the whole screen and then dial that back with `zoomBackOff`? For each `1` unit increase in whatever we call the variable, the viewport is downsized by a factor of 2.

Reply:
I'm ok with the variable name, but we need a better description